micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/FileSystemPollerTest.groovy

```diff
@@ -39,23 +43,42 @@ class FileSystemPollerTest extends AbstractIntegrationTest {
         poller = context.getBean(FileSystemPoller)
     }
 
-    def 'should reload configuration once when file system touched'() {
+    @Unroll
+    def 'should reload configuration once when file #configFile from config location with allowed name is touched'() {
         when:
-            oneConfigurationFileWasChanged("micro-app.yaml")
+            oneConfigurationFileWasChanged(configFile)
+
         then:
-            conditions.eventually {
-                counter.value == old(counter.value) + 1
-            }
-            counter.value == old(counter.value) + 1
+            int oldValue = old(counter.value)
+            waitForCounterIncrement(oldValue)
+            assertThatCounterWasIncrementedOnce(oldValue)
```

Comment:
I'm not sure if after refactoring it is more readable (again it is a matter of taste, anyone else?)